pkg/provider/cloud/kubevirt/network_policy.go

```diff
@@ -39,6 +40,11 @@ func clusterIsolationNetworkPolicyCreator() reconciling.NamedNetworkPolicyCreato
 							{
 								PodSelector: &metav1.LabelSelector{},
 							},
+							{
+								NamespaceSelector: &metav1.LabelSelector{
```

Comment:
Do we want to restrict to `cdi-upload` pods also ?

Reply:
It can be done to restrict to `cdi-upload` pods.